release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -117,6 +117,12 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-14-cert-manager-operator-1-11-1"]
+=== cert-manager Operator general availability
+
+{cert-manager-operator} was introduced as a Technology Preview in {product-title} 4.11 and is now generally available in {product-title} 4.14.
```

Comment:
Hmm, if I'm a new reader who does not know more about, this wording still make me not clear: is it TP or GA in 4.12 ~ 4.13? And the preview hides 1.11 now, this wording makes others not clear: 1.10 also GA in 4.14?
How about below:
```suggestion
{cert-manager-operator} 1.11.1 is now generally available in {product-title} 4.14 as well as 4.12 ~ 4.13.
```

Reply:
+1

Reply:
Addressed